### PR TITLE
Port default print layout

### PR DESCRIPTION
**Pull request by llienher**
Port 2.4 features in master branch

### CODE REVIEW

**Comment by llienher** [comment]
@sbrunner I have an issue in the console, testing this port:

```
vendor.js?dev:2 org.mapfish.print.ExtraPropertyException: Extra properties found in the request attributes
Extra Properties: 

	* goodnessOfFit

All allowed properties are: 

	* northArrow
	* comments
	* debug
	* requestHeaders
	* legend
	* datasource
	* title
	* map
	* scalebar
```

Do you have an idea what is missing ? I'm pretty sure we already had this problem in the past but cannot figure out what we did to fix it...

**Comment by sbrunner** [comment]
@llienher Where do you found that?

**Comment by llienher** [comment]
@sbrunner It's in my console, error when trying to print

**Comment by sbrunner** [comment]
It's not already present on master?

**Comment by llienher** [comment]
Yes, it is actually the previous merged PR that broke that, not related to this one. So visible on master.

**Comment by sbrunner** [comment]
C'est aussi le cas sur 2.4 :-(

**Comment by sbrunner** [comment]
This line looks wrong: https://github.com/camptocamp/ngeo/pull/5621/files#diff-253425453dea4e54ec1f5fde0c68b1d5R1000
Shouldn't it be removed?

**Comment by llienher** [comment]
@sbrunner Nice catch, seems to fix it if we remove it.

**Comment by sbrunner** [comment]
But now this line will not work...
https://github.com/camptocamp/ngeo/pull/5621/files#diff-8ba1e081de38ec354b188acb2fea6454R144

File: src/print/Service.js [hunk 1]
```diff
@@ -163,7 +163,8 @@ PrintService.prototype.createSpec = function(
     rotation: customAttributes.rotation
   });
 
-  this.goodnessOfFit_ = customAttributes['goodnessOfFit'];
+  this.goodnessOfFit_ = customAttributes.goodnessOfFit;
+  delete customAttributes.goodnessOfFit;
```
Comment (sbrunner) [inline src/print/Service.js]: This is a big hack, it shouldn't be passed throw the customAttributes, but throw a new optional? attribute.

**Comment by llienher** [comment]
@sbrunner I did pass the parameter to the service from the component via an optional parameter in the createSpec method. That's cleaner. :)

**Pull request merged**